src/50_Samples_%26_Templates/Product.html

```diff
@@ -269,62 +339,226 @@ <h4 class="category"><a href="/samples_templates/product_listing/preview/">More<
+
+    We bind the `text` attribute to the
+      value of expression `[text]="product[selectedColor || 'green'].sizes[selectedSize || 'S']"`.
+      `product` is the id  of the `amp-state` json. We use green and S as default values for the
+      variables `selectedColor` and `selectedSize`.
+
+    Notice how we set a default value for the price: values defined inside `amp-state`
+      will not be accessed at the page load.
+ -->
+      <p class="price-description">Price:
+        <span [text]="product[selectedColor || 'green'].sizes[selectedSize || 'S']">$1.99</span>
+    </p>
+    <!-- -->
+    <a name="product-configuration-anchor"></a>
+
     <!-- #### Product Configuration -->
-    <!-- The add to cart action is implemented using  [amp-form](/components/amp-form/). In our sample we use the [amp-selector](/components/amp-selector/) for selecting different product properties. Pressing the `ADD TO CART` button adds the product to a shopping cart page using the properties you have selected. Notice that the button URL contains the query `clientId={{ClientId}}`. If an item is successfully added to the shopping cart, we redirect the user to the shopping cart page. The redirect target is configured via a header in the server's form response ([redirecting after submission](https://github.com/ampproject/amphtml/blob/master/extensions/amp-form/amp-form.md#redirecting-after-a-submission)):
+    <!--
+    We use [amp-selector](/components/amp-selector/) together with [amp-bind](/components/amp-bind/) for selecting different product properties and to update the page for showing the price, the size availability, the gallery and preview of images.
 
-      access-control-expose-headers:AMP-Access-Control-Allow-Source-Origin,AMP-Redirect-To
-      amp-redirect-to:http://ampbyexample.com/shopping_cart/?clientid=amp-123456789
+    Every time you select a color, we set the expression `selectedColor=event.targetOption`.
+    This is triggering all the attributes and elements bound to `selectedColor` to update:
+    the price (bound with the attribute `[text]`), the gallery, the image preview and
+    the size selector (all bound with `[class]`).
 
-    We are using the `CLIENT_ID` variable to identify the user, this enables storing a shopping cart between repeated visits of an AMP page (either via the AMP Cache or the original host). This variable can be used inside an amp-form by declaring an hidden input value with `default-value="CLIENT_ID(cart)`. Read more about variable substitution [here](https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md). -->
+    The add-to-cart action action is implemented using  [amp-form](/components/amp-form/).
+    Pressing the `ADD TO CART` button adds the product to a shopping cart page using the properties you have selected. Notice that the button URL contains the query `clientId={{ClientId}}`.
```

Comment:
' Notice that the button URL contains the query `clientId={{ClientId}}`.' how can the reader notice this without seeing the code?